salt/modules/saltutil.py

```diff
@@ -785,6 +801,24 @@ def kill_job(jid):
     return signal_job(jid, salt_SIGKILL)
 
 
+def kill_all_jobs():
+    '''
+    Sends a kill signal (SIGKILL 9) to all currently running jobs
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' saltutil.kill_all_jobs
+    '''
+    # Some OS's (Win32) don't have SIGKILL, so use salt_SIGKILL which is set to
+    # an appropriate value for the operating system this is running on.
+    ret = []
+    for data in running():
+        ret.append(signal_job(data['jid'], salt_SIGKILL))
```

Comment:
I only copied the signal from kill_job(jid) which references a "nasty bug in Win32 python. For sigterm, that was not needed according to term_job(jid)